lib/core/include/irods/irods_path_recursion.hpp

```diff
@@ -23,14 +23,14 @@ namespace irods
     // Returns true if the user path is a directory which has not
     // yet been examined, or, throws an irods::exception if the path has
     // already been examined (it's in the set<> already).
-    // The bool parameter is true if the "--link" flag was specified
+    // The bool parameter is true if the "--ignore-symlinks" flag was specified
     bool is_path_valid_for_recursion(boost::filesystem::path const &, recursion_map_t &, bool);
```

Comment:
Please rephrase

Reply:
Some possible rephrasings:

- `The bool parameter signifies if the "--ignore-symlinks" flag was given.`
- `The "--ignore-symlinks" flag sets the bool parameter, which is then used when processing symbolic links.`
- `The final parameter of this function is set based on the presence or absence of "--ignore-symlinks". This is then used in determining whether or not a symbolic link should be followed. See the function implementation for more details.`

Reply:
> `When the last parameter is set to true, symbolic links are not processed. Otherwise, they are processed conditionally. See implementation for more details.`

That sounds good to me.

Reply:
Done.